### DISCUSSION TITLE
detach_window to next or prev tab?

### DISCUSSION THREAD

**Discussion by leandwo** [kovidgoyal/kitty#3877] [Q&A]
Is there a way to send a window to the next or prev tab?

It looks like the only options on detach_window are `new`, `new-tab`, and `ask`

**Comment by kovidgoyal** [comment]
Use ask. Or send a PR adding tab-left, tab-right, tab-prev and tab-next.

**Reply by leandwo** [reply]
I'll see if I can add it.

**Reply by kovidgoyal** [reply]
I have implemented it, but not tested it, I leave that to you. https://github.com/kovidgoyal/kitty/commit/17ff317d3092b3c7f18c2c46569a39457abb4ba6
